runtime/compiler/env/j9method.cpp

```diff
@@ -3790,6 +3790,12 @@ void TR_ResolvedJ9Method::construct()
       {  TR::unknownMethod}
       };
 
+   static X SingleByteDecoderMethods[] =
+      {
+      {x(TR::sun_nio_cs_SingleByteDecoder_decodeToLatin1Impl,      "decodeToLatin1Impl", "([BII[BI[B)I")},
```

Comment:
We should add comment like above - to highlight that this method is used in Semeru Rutime for z/OS.

Reply:
Added comment in bda4b2721e47d238dfdc5bb9c45c9247ac57feb0